model/src/main/java/org/gorpipe/gor/util/SqlReplacer.java

```diff
@@ -7,24 +7,11 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
-// TODO: Should be synced with CommandSubstitutions.
```

Comment:
I don't know if I understand what this comment was suggesting, but did these changes accomplish that?